src/ext.rs

```diff
@@ -167,37 +169,25 @@ pub fn block_hash(block_number: u64) -> H256 {
 	res
 }
 
-unsafe fn fetch_address<F>(f: F) -> Address where F: Fn(*mut u8) {
-	let mut res = Address::zero();
-	f(res.as_mut_ptr());
-	res
-}
-
-unsafe fn fetch_u256<F>(f: F) -> U256 where F: Fn(*mut u8) {
-	let mut res = [0u8; 32];
-	f(res.as_mut_ptr());
-	U256::from_big_endian(&res)
-}
-
-/// Get the block’s beneficiary address (i.e miner's account address).
+/// Get the block’s beneficiary address (i.e miner's account address)
```

Comment:
This term has different names and much better description in other sources, there should be short summation of those